internal/restic/node_windows.go

```diff
@@ -358,7 +361,20 @@ func (node *Node) fillGenericAttributes(path string, fi os.FileInfo, stat *statT
 		// Also do not allow processing of extended attributes for ADS.
 		return false, nil
 	}
-	if !strings.HasSuffix(filepath.Clean(path), `\`) {
+	if strings.HasSuffix(filepath.Clean(path), `\`) {
```

Comment:
This approach won't work for backups using relative paths.
```
cd /D Z:   # some network share
restic backup local/file
```

Reply:
Thanks, I believe I've addressed this now.